sources/PPU/Background.hpp

```diff
@@ -7,16 +7,93 @@
 
 #include <stdint-gcc.h>
 #include <array>
+#include <vector>
 #include "../Models/Vector2.hpp"
+#include "../Ram/Ram.hpp"
+#include "PPU.hpp"
 
-class Background {
-	int width;
-	int height;
-	int bpp;
-	bool directColor;
-	bool highRes;
-	std::array<std::array<uint16_t, 1024>, 1024> buffer;
-};
+//! @brief Transform SNES color code BGR to uint32_t RGB
+uint32_t getRealColor(uint16_t color);
 
+namespace ComSquare::PPU
+{
+	class PPU;
+	class Background {
+	#define NB_CHARACTER_WIDTH 32
```

Comment:
Do not use define for constants, use a constexpr. Define are not scopped by namespaces, constexpr are.